blockly-core/core/utils/xml.js

```diff
@@ -17,6 +17,12 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
+
+// Ignore "Expected a conditional expression and instead saw an
+// assignment" warnings.
+/* jshint -W084 */
```

Comment:
It's probably worth having a larger discussion about what code standards we want to apply to blockly. The code itself was clearly originally written using a very different styleguide. Moving forward, do we want to enforce our style on this code, or the original style?